x-pack/plugins/monitoring/server/kibana_monitoring/collectors/get_ops_stats_collector.js

```diff
@@ -6,11 +6,12 @@
 
 import { KIBANA_STATS_TYPE } from '../../../common/constants';
 import { opsBuffer } from './ops_buffer';
+import { sourceKibana } from '../lib';
 
 /*
  * Initialize a collector for Kibana Ops Stats
  */
-export function getOpsStatsCollector(server) {
+export function getOpsStatsCollector(server, kbnServer) {
```

Comment:
`kbnServer.server` is the same as the `server` argument here, so if you wanted you could reduce these argument lists to just `(kbnServer)`

Reply:
Good to know! I think I'd like to handle this in a later change though. Ok if I just add a TODO comment for now?